src/hooks/useCart/useCart.tsx

```diff
@@ -235,10 +236,19 @@ export const useCart = (
    */
   const _completeCheckout = useCallback(() => {
     const complete = async (): Promise<void> => {
+      const allCleanedIdentifierInputs: IdentifierInput[] = [];
       const transactions = Object.values(cart)
         .filter(({ quantity }) => quantity)
         .map(({ category, quantity, identifierInputs }) => {
-          return { category, quantity, identifierInputs };
+          const cleanedIdentifierInputs = cleanIdentifierInputs(
+            identifierInputs
+          );
+          allCleanedIdentifierInputs.push(...cleanedIdentifierInputs);
```

Comment:
looks like we no longer need `allCleanedIdentifierInputs` here since it's not validated in the `completeCheckout` fn!

Reply:
removed!